platform/app/cypress/support/aliases.js

```diff
@@ -26,7 +26,7 @@ export function initCommonElementsAliases(skipMarkers) {
   cy.get('[data-cy="trackedMeasurements-btn"]').click();
 
   // TODO: Panels are not in DOM when closed, move this somewhere else
-  cy.get('[data-cy="trackedMeasurements-panel"]').as('measurementsPanel');
```

Comment:
This change should test the trackedMeasurements-panel here.  If that means passing down an extra parameter that indicates the panel name, then it will have to be done.  That also allows different panels to have different names.